akka-http/src/main/java/akka/http/javadsl/coding/Coder.java

```diff
@@ -25,9 +29,40 @@
         this.underlying = underlying;
     }
 
+    public HttpResponse encodeMessage(HttpResponse message) {
```

Comment:
Ah, I see. You can add `def transformEntityDataBytes[T](transformer: Graph[FlowShape[ByteString, ByteString], T]): Self` to `scaladsl.HttpMessage` which will be implemented by the concrete implementations in `HttpRequest` and `HttpResponse`. Then